components/authz-service/README.md

```diff
@@ -621,10 +630,13 @@ To add a new default policy, the following is needed:
 
 1. A new IAM v1 policy: these are defined in [migrations](storage/postgres/migration/sql/),
    and the [v1 constants package](constants/v1/constants.go#L3-L4).
-2. A new IAM v2 (system) policy: they are defined in [server/v2/system](server/v2/system.go#L31).
-3. A new IAM v2 default policy (which can be deleted by users), or any additions
-   to default roles, are done in [datamigrations](storage/postgres/datamigration/sql/).
-3. Migration logic: When upgrading from IAM v1 to v2 (v2.1), v1 policies are
+1. You will need to add ONE of the following (ask auth team if you're not sure which):
+
+- A new IAM v2 (system) policy: they are defined in [server/v2/system](server/v2/system.go#L31).
+- A new IAM v2 default policy (which can be deleted by users), or any additions
+    to default roles, which are done in [datamigrations](storage/postgres/datamigration/sql/).
+
+1. Migration logic: When upgrading from IAM v1 to v2.1, v1 policies are
```

Comment:
I would urge using actual numbering instead of all "1"s (like the lines you deleted). Both are valid, but with a sequence you can have the context while reading the "raw" file; you do not have to render it.

Reply:
Due to the nested list I'm not able to do this without adding lint errors so I'm going to leave it as it.